lib/js/jsconsole.nim

```diff
@@ -67,4 +69,15 @@ proc timeLog*(console: Console, label = "".cstring) {.importcpp.}
 proc table*(console: Console) {.importcpp, varargs.}
   ## https://developer.mozilla.org/docs/Web/API/Console/table
 
+since (1, 5):
+  # proc nor func can not be used here, because may eval assertion at compile-time,
+  # writing directly false or true to the JavaScript instead of the expression.
+  # importjs nor importcpp can not be used with template.
+
+  template doAssert*(console: Console; assertion) =
```

Comment:
rename it (jassert or similar), and make it clear in doc comment and runnableExamples that assert failure keeps going and just prints to console; for nodejs, specify whether it prints to stdout or stderr.